### PR TITLE
(test): Add test for `patient-list-table.component.tsx`

### PR DESCRIPTION
**Pull request by ayush-AI**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

In this PR, I have added test for `patient-list-table.component.tsx`

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by ayush-AI** [comment]
cc/ @Piumal1999 @anjula-sack @jayasanka-sack

File: packages/esm-patient-list-app/src/patient-list-list/patient-list-table.test.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      paginated: false,
+    });
+
+    render(<PatientListTableContainer patientLists={patientLists} listType={''} headers={header} />);
+    expect(screen.getByRole('table')).toBeInTheDocument();
+    await waitFor(() => {
+      expect(screen.getByText('My lists')).toBeInTheDocument();
+    });
+
+    // to Enquire why the data is not coming and add assertion to check the data
```
Comment (Piumal1999) [inline packages/esm-patient-list-app/src/patient-list-list/patient-list-table.test.tsx:98]: Hi @ayush-AI , Can you elaborate on the comments on this file?

Comment (ayush-AI) [reply packages/esm-patient-list-app/src/patient-list-list/patient-list-table.test.tsx:98]: Means the Patient lists is not showing up.

Comment (ayush-AI) [reply packages/esm-patient-list-app/src/patient-list-list/patient-list-table.test.tsx:98]: We will enquire about this later.

Comment (ayush-AI) [reply packages/esm-patient-list-app/src/patient-list-list/patient-list-table.test.tsx]: I have added the comment as TODO. Please check @Piumal1999

**Pull request merged**